src/builders/scene/addEnvironment.ts

```diff
@@ -0,0 +1,29 @@
+import { Scene, WebGLRenderer } from "three"
+
+import { PMREMGenerator } from "three/examples/jsm/pmrem/PMREMGenerator"
+import { PMREMCubeUVPacker } from "three/examples/jsm/pmrem/PMREMCubeUVPacker"
+import { EquirectangularToCubeGenerator } from "three/examples/jsm/loaders/EquirectangularToCubeGenerator"
+import GameFieldAssets from "../../loaders/scenes/GameFieldAssets"
+
+export const addEnvironment = (scene: Scene, renderer: WebGLRenderer) => {
+  const { environment } = GameFieldAssets.getAssets()
+
+  const cubeGenerator = new EquirectangularToCubeGenerator( environment, { resolution: 2048 } );
```

Comment:
I doubt this has any impact in the grand scheme of things but we're using a resolution of 512 in the shadows so we might want to pull this out into a constant that we can modify based on the user's platform. I meant to do this a long time ago but have preoccupied